spharpy/classes/sh.py

```diff
@@ -6,7 +6,108 @@
 import spharpy as sy
 
 
-class SphericalHarmonics:
+class SphericalHarmonicDefinition:
+    """Class storing the definition of spherical harmonics.
+
+    Attributes
+    ----------
+    condon_shortley : bool
```

Comment:
Does not document `condon_shortley = 'auto'`. Maybe not needed it class would be private, but nice to have in any case.